login.php

```diff
@@ -25,11 +25,11 @@
 setCurrentSchoolYear($guid, $connection2);
 
 //The current/actual school year info, just in case we are working in a different year
-$_SESSION[$guid]['gibbonSchoolYearIDCurrent'] = $_SESSION[$guid]['gibbonSchoolYearID'];
-$_SESSION[$guid]['gibbonSchoolYearNameCurrent'] = $_SESSION[$guid]['gibbonSchoolYearName'];
-$_SESSION[$guid]['gibbonSchoolYearSequenceNumberCurrent'] = $_SESSION[$guid]['gibbonSchoolYearSequenceNumber'];
+$gibbon->session->setet('gibbonSchoolYearIDCurrent', $gibbon->session->get('gibbonSchoolYearID'));
```

Comment:
It looks like these lines ended up formatted incorrectly, perhaps a regex replace gone awry.

Reply:
That's fixed now.